### ISSUE TITLE
Cannot use AMD GPU

### ISSUE THREAD

**Issue by kylewill0725** [exploitagency/vanitygen-plus#6]
Tried using a R9 280x to get an address for myself but kept getting this error:

X:\Users\kylew\Downloads\windows-binary>oclvanitygen.exe -v -o keys.txt -p 1 -d 0 1111 1112 1113 1114 1115
Prefix difficulty:             16777216 1111
Prefix difficulty:               364829 1112
Prefix difficulty:              4553521 1113
Prefix difficulty:              4553521 1114
Prefix difficulty:              4553521 1115
Next match difficulty: 289064 (5 prefixes)
Device: Tahiti
Vendor: Advanced Micro Devices, Inc. (1002)
Driver: 1912.5 (VM)
Profile: FULL_PROFILE
Version: OpenCL 1.2 AMD-APP (1912.5)
Max compute units: 32
Max workgroup size: 256
Global memory: -1073741824
Max allocation: -2080374784
OpenCL compiler flags: -DDEEP_PREPROC_UNROLL -DVERY_EXPENSIVE_BRANCHES -DDEEP_VLIW
Compiling kernel, can take minutes...failure.
clBuildProgram: CL_BUILD_PROGRAM_FAILURE
Build log:
"X:\Users\kylew\AppData\Local\Temp\OCL13948T1.cl", line 173: error: expected
          an expression
  __constant bignum bn_zero = {};
                               ^

1 error detected in the compilation of "X:\Users\kylew\AppData\Local\Temp\OCL13948T1.cl".
Frontend phase failed compilation.
Available OpenCL platforms:
0: [Intel(R) Corporation] Intel(R) OpenCL
    0: [Intel(R) Corporation] Intel(R) HD Graphics 4600
    1: [Intel(R) Corporation] Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
1: [Advanced Micro Devices, Inc.] AMD Accelerated Parallel Processing
    0: [Advanced Micro Devices, Inc.] Tahiti
    1: [GenuineIntel] Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz

X:\Users\kylew\Downloads\windows-binary>

**Comment by Rav3nPL** [comment]
Same there
```
Compiling kernel, can take minutes...failure.
clBuildProgram: CL_BUILD_PROGRAM_FAILURE
Build log:
"C:\Users\vg\AppData\Local\Temp\OCL1972T1.cl", line 173: error: expected an
          expression
  __constant bignum bn_zero = {};
                               ^
```
win10x64 Radeon HD7970 drivers: 17.1.1

**Comment by exploitagency** [comment]
Thanks for the fix, I am releasing version 1.42 binary soon to include it.  Already merged changed to the main source.

Reopen the issue if this patch does not solve the problem for you.  Be sure to delete any oclbin files that may have been created.  I do not believe there will be one there though in the case of a failure to compile.
